provisioning/README.md

```diff
@@ -23,8 +23,32 @@ You'll need to create you openshift router and registry (see https://github.com/
 
 ## Fully Automated Environment Provisioning
 
-See sub-directories for environment specific provisioning scripts and instructions.
+We now support full end-to-end Environment Provisioning.
 
 Current implementations:
 
  - Openstack
+
+### Instructions
+
+1. [Configure client tools](provisioning/openstack/README.md)
+2. Clone this repo.
+3. Run provisioning script:
+```bash
+# Run osc-provision with no options to show Usage output and options
```

Comment:
Since running with no options generates an error (i.e. the "Missing argument" error), I'd suggest telling the user to use -h or --help instead (if these options are available)